module/VuFind/config/module.config.php

```diff
@@ -460,7 +460,11 @@
             'Zend\Validator\Csrf' => 'VuFind\Validator\Csrf',
         ],
     ],
-    'translator' => [],
+    'translator_plugins' => [
+        'factories' => [
+            'VuFind\I18n\Translator\Loader\ExtendedIni' => 'Zend\ServiceManager\Factory\InvokableFactory',
+        ]
+    ],
```

Comment:
As the contents of `translator_plugins` will be merged into `Zend\I18n\Translator\LoaderPluginManager` configuration we are able to override default loaders as well as provide additional loaders by configuration.